# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bluez-hciemu/bluez-libs-2.18.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./hciemu.c.orig	2005-11-01 16:00:54.000000000 -0300
+++ ./hciemu.c	2005-11-01 16:02:46.000000000 -0300
@@ -555,8 +555,8 @@
 		command_complete(ogf, ocf, sizeof(ln), &ln);
 		break;
 
-	case OCF_WRITE_PG_TIMEOUT:
-	case OCF_WRITE_CA_TIMEOUT:
+	case OCF_WRITE_PAGE_TIMEOUT:
+	case OCF_WRITE_CONN_ACCEPT_TIMEOUT:
 		status = 0x00;
 		command_complete(ogf, ocf, 1, &status);
 		break;
